utils/property_map.go

```diff
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+)
+
+type PropertyMap map[string]interface{}
+
+//Value - simply returns the JSON-encoded representation of the struct.
+func (p PropertyMap) Value() (driver.Value, error) {
```

Comment:
Marshalling of JSON can be done directly wherever it is needed, do we need this as a separate function?

Also, We don't seem to be using this function anywhere 🤔 
We could add functions to the code base as and when we use it 🤔

Reply:
I use it for handling JSONB property...please [refer](https://coussej.github.io/2016/02/16/Handling-JSONB-in-Go-Structs/) here